libsignal-service/src/push_service.rs

```diff
@@ -221,14 +221,17 @@ impl PreKeyResponseItem {
     ) -> Result<PreKeyBundle, SignalProtocolError> {
         PreKeyBundle::new(
             self.registration_id,
-            self.device_id,
+            self.device_id.into(),
```

Comment:
I could propage the strong types on the service level. But this would be breaking change then. WDYT?

Reply:
I don't think we really shy away from breaking the interface, especially since it broke in libsignal-client already. Would you be okay with me merging #156 first, then rebase this on top?